data/presets/railway/yard.json

```diff
@@ -2,6 +2,7 @@
     "icon": "temaki-freight_car",
     "fields": [
         "name",
+        "railway/ref",
```

Comment:
PR description mentions adding `railway:ref` to [railway=junction](https://osm.wiki/Tag:railway=junction) - not railway yard

Reply:
I am not opposed to it, just got confused when comparing code and description